app/auth/__init__.py

```diff
@@ -0,0 +1,30 @@
+from flask import Blueprint
+from app import oauth
+
+auth = Blueprint('auth', __name__)
+
+facebook = oauth.remote_app(
+    'facebook',
+    consumer_key='449699711888450',
+    consumer_secret='a968e17aeffb7d0c35ff22d5c7fc5a7f',
```

Comment:
Security convention requires that you use system variables to store passwords and keys. Use a system variables and update your readme. Good job!